multiuser/keycloak/che-multiuser-keycloak-ide/src/main/java/org/eclipse/che/multiuser/keycloak/ide/Keycloak.java

```diff
@@ -21,6 +21,16 @@ protected Keycloak() {
     super();
   }
 
+  public static native boolean isMultihostEnvironment() /*-{
+    try {
+      var dummy = $wnd.parent['_keycloak'];
```

Comment:
dummy var unused here

Reply:
@vparfonov Just for checking here if `$wnd.parent['_keycloak']` is readable.
What is the best way to check it?